foundations/javascript_basics/object_basics.md

```diff
@@ -20,6 +20,74 @@ Objects are a *very* important part of the JavaScript language, and while for th
 1. This JavaScript.info [article on objects](https://javascript.info/object) is the best place to get started.
 1. The [MDN tutorial on objects](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Basics) isn't bad either, so check it out if you need another take on the subject.
 
+#### Differences between objects and primitives
+
+Earlier in the curriculum you've learned about [primitive data types](https://www.theodinproject.com/lessons/foundations-data-types-and-conditionals). Now you've seen the object data type, the main difference between the two is that primitives can contain only a single thing (string, number, etc). Objects are used to store collection of data and more complex entities.
+Besides the formal differences, there are also some technical differences, which affect how we use each data type.
+When you define a primitive variable, it will contain a copy of the information provided to it:
```

Comment:
Regarding the formatting I'll change it no problem.
Regarding this:
> Now you've seen the object data type
Maybe I phrased it incorrectly, but I was trying to say that previously the learner have seen the primitive types (the link), and **now** (a section above) he have seen the object type. Regarding the data type maybe something like this would work?:
> Now you've seen the object **data type**, which includes but not limited to, objects (`{key: value}`), arrays and functions. The main difference between the two is that primitives can contain only a single thing (string, number, etc). Objects are used to store collection of data and more complex entities.